modules/nw-ingress-controller-nodeportservice-projects.adoc

```diff
@@ -0,0 +1,161 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/nw-configuring-ingress-controller-endpoint-publishing-strategy.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-controller-nodeportservice-projects_{context}"]
+= Adding a single NodePort service to an Ingress Controller
+
+Instead of creating a `NodePort`-type `Service` for each project, you can create a custom Ingress Controller to use the `NodePortService` endpoint publishing strategy. To prevent port conflicts, consider this configuration for your Ingress Controller when you want to apply a set of routes, through Ingress sharding, to nodes that might already have a `HostNetwork` Ingress Controller.  
+
+Before you set a `NodePort`-type `Service` for each project, read the following considerations:
+
+* You must create a wildcard DNS record for the Ingress Controller. For more information about the required DNS records for routes, see "User-provisioned DNS requirements".
+* You must expose a route for your service and specify the `--hostname` argument for your custom Ingress Controller domain.
+* Ensure that all Ingress Controller routes can access your {product-title} cluster through the host IP address assigned to each node.
```

Comment:
```suggestion
~~
* Ensure that all Ingress Controller routes can access your {product-title} cluster through the host IP address assigned to each node.
~~
```

Reply:
I would leave this line out.  It is now incorporated with my suggestion in the DNS record.

Reply:
Removed:

> * Ensure that all Ingress Controller routes can access your {product-title} cluster through the host IP address assigned to each node.